cli/src/args.rs

```diff
@@ -47,6 +48,14 @@ pub struct CompileCommand {
     /// Opens the output file after compilation using the default PDF viewer
     #[arg(long = "open")]
     pub open: Option<Option<String>>,
+
+    #[arg(long, short, default_value_t = false)]
+    pub image: bool,
```

Comment:
I would prefer a target option like `-t png` which could be backed by an enum of possible targets, each with individual options. Not sure how well this maps to clap.